expression/gval.go

```diff
@@ -3,13 +3,29 @@ package expression
 import (
 	"context"
 	"fmt"
+	"go.uber.org/zap"
+	"sync"
 	"time"
 
 	"github.com/PaesslerAG/gval"
 	ttlcache "github.com/myrteametrics/myrtea-sdk/v5/cache"
 )
 
+type GlobalVariables struct {
+	listKeyValue   map[string]interface{}
+	listKeyValueMu sync.RWMutex
+}
+
+type MergedVariables struct {
+	globalVars *GlobalVariables
+	localVars  map[string]interface{}
+}
+
+const prefixGlobalVars = "global_variable_"
+
 var (
+	GlobalVars *GlobalVariables = &GlobalVariables{listKeyValue: make(map[string]interface{})}
```

Comment:
```suggestion
	globalVars *GlobalVariables = &GlobalVariables{listKeyValue: make(map[string]interface{})}
```

this variable can be private ?

Reply:
No, unless I create a public function, the goal is to be able to update the global variable from the engine.